src/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -237,22 +237,23 @@ var resetResponse = function (interaction) {
  * @param {Object} interaction - the extended text interaction model
  * @param {object} response
  */
-var setResponse = function (interaction, response) {
-    var _setMultipleVal = function (identifier, value) {
-        interaction
-            .getContainer()
-            .find('#' + identifier)
-            .val(value);
+const setResponse = (interaction, response) => {
+    const _setMultipleVal = (identifier, value) => {
+        interaction.getContainer().find(`#${identifier}`).val(value);
     };
 
-    var baseType = interaction.getResponseDeclaration().attr('baseType');
+    const baseType = interaction.getResponseDeclaration().attr('baseType');
 
-    if (response.base && response.base[baseType] !== undefined) {
+    if (response.base === null && Object.keys(response).length === 1) {
+        response = { base: { string: '' } };
+    }
+
+    if (response.base && typeof response.base[baseType] !== 'undefined') {
```

Comment:
Do you need the first condition? `response.base`. As I know, `null` is the only one that is `false`, but if it would be `null`, you would set it to string in the previous condition.

Reply:
Well, you are right: if `response.base` is null it may be replaced by the previous condition, if this is the only one property. However, this will not work if it is not defined but `response.list` is set instead.